resources/client_scripts/errorHandling.js

```diff
@@ -0,0 +1,15 @@
+const debug = window.debug;
+localStorage.debug = 'team-link:*';
```

Comment:
I don't think that we should always enable the debug output from our code. I can always put a value to the local storage via DevTools console and just reload the page.